src/main/resources/swagger.api/requests-mediated.yaml

```diff
@@ -118,10 +118,29 @@ paths:
           $ref: '#/components/responses/notFoundResponse'
         '500':
           $ref: '#/components/responses/internalServerErrorResponse'
+  /requests-mediated/confirm-item-arrival:
+    post:
+      description: Confirm item arrival
+      operationId: confirmItemArrival
+      tags:
+        - mediatedRequest
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/confirmItemArrivalRequest'
+        required: true
+      responses:
+        '400':
+          $ref: '#/components/responses/badRequestResponse'
+        '500':
+          $ref: '#/components/responses/internalServerErrorResponse'
```

Comment:
We're only specifying 400 and 500 statuses here, but in the controller we return 200:
`return ResponseEntity.ok(null);`

Reply:
Can you suggest the line for 200 when we return void?

Reply:
updated